### PR TITLE
Microverse Project : First Hello Microverse Project

### PR DESCRIPTION
**Pull request by PremChand12**
## First Microverse Project : -
     
      In this project : 
          -Added HTML and CSS files.
          -Added Linters 
          -Created New Branches

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,238 @@
...
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  
+  <br/>
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
```
Comment (usorfaitheloho) [inline README.md]: Kindly remove all commented code from your README file to enable you have a clean code that is easily maintainable that is also easy to understand 😄

File: README.md [hunk 2]
```diff
@@ -1 +1,238 @@
...
+In order to run this project you need to clone github repository using command : git clone https://github.com/PremChand12/hello-world-microverse
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
```
Comment (usorfaitheloho) [inline README.md]: Kindly remove all commented code from your README file to enable you have a clean code that is easily maintainable that is also easy to understand 😄

File: README.md [hunk 3]
```diff
@@ -1 +1,238 @@
...
+Clone this repository to your desired folder: cd my-folder
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
```
Comment (usorfaitheloho) [inline README.md]: Kindly remove all commented code from your README file to enable you have a clean code that is easily maintainable that is also easy to understand 😄

File: README.md [hunk 4]
```diff
@@ -1 +1,238 @@
...
+Install this project with: cd my-project
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
```
Comment (usorfaitheloho) [inline README.md]: Kindly remove all commented code from your README file to enable you have a clean code that is easily maintainable that is also easy to understand 😄

File: README.md [hunk 5]
```diff
@@ -1 +1,238 @@
...
+
+To run the project, use code editor e.g Visual Studio
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
```
Comment (usorfaitheloho) [inline README.md]: Kindly remove all commented code from your README file to enable you have a clean code that is easily maintainable that is also easy to understand 😄

File: README.md [hunk 6]
```diff
@@ -1 +1,238 @@
...
+
+To run tests, use Visual Studio Live Server
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
```
Comment (usorfaitheloho) [inline README.md]: Kindly remove all commented code from your README file to enable you have a clean code that is easily maintainable that is also easy to understand 😄

File: README.md [hunk 7]
```diff
@@ -1 +1,238 @@
...
+git commit and,
+git push
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
```
Comment (usorfaitheloho) [inline README.md]: Kindly remove all commented code from your README file to enable you have a clean code that is easily maintainable that is also easy to understand 😄

File: README.md [hunk 8]
```diff
@@ -1 +1,238 @@
...
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
```
Comment (usorfaitheloho) [inline README.md]: Please kindly add features that will be implemented in the future,so a user knows what to be expected in the future 😄

File: README.md [hunk 9]
```diff
@@ -1 +1,238 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
```
Comment (usorfaitheloho) [inline README.md]: The FAQ is the only optional section of the README file so when you don't have anything to write there, please remove it 👍

File: README.md [hunk 10]
```diff
@@ -1 +1,238 @@
...
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (usorfaitheloho) [inline README.md]: Kindly add an MIT file and attach the link here as your link is broken here, so as to enable you have a clean code 👍

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @PremChand12  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 11]
```diff
@@ -1 +1,147 @@
...
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
```
Comment (brainconnect93) [inline README.md]: - [ ] [Optional] Kindly remove this section as it is not needed in your readme file. You already have the table of content below.

File: README.md [hunk 12]
```diff
@@ -1 +1,147 @@
...
+I want to thank my coding partner Nouman Walli who discussed the technical areas with me.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project has no license.
+
```
Comment (brainconnect93) [inline README.md]: - [ ] As suggested by the previous reviewer, Kindly add the MIT license link to your readme file as this serves as a copywrite that gives the visitor of your page access to make use of the content you have on your page.
Add the MIT file to your root directory and target it in your readme file

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @PremChand12, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 

- Descriptive proper Pull Request summary. ✔️  
- Hello Microverse project done ✔️ 
- There is a `.gitignore` file listing `test.md` file and `node_modules` ✔️
- Good use of GitHub flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @PremChand12, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by PremChand12** [comment]
Thank you so much for the great news!

**Pull request merged**